types/supports-hyperlinks/index.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module 'supports-hyperlinks' {
```

Comment:
[note] `supports-hyperlinks` doesn't have type definitions (even on `@types/`).

https://www.npmjs.com/package/supports-hyperlinks

Reply:
Curious, is this something that would be helpful to contribute upstream? Looking at the package, it doesn't seem too complicated - maybe I can give it a shot?

Reply:
@mattxwang That would be great! 👍🏼

Reply:
Sounds good - I can try to take a crack at it next weekend; probably shouldn't prevent this PR from being merged!

Reply:
Have created https://github.com/jamestalmage/supports-hyperlinks/issues/15!

Reply:
Looks like this may have been resolved with https://github.com/jamestalmage/supports-hyperlinks/pull/21?

Reply:
@mattxwang Thanks for sharing the news! We can delete our custom type definitions with [`supports-hyperlinks@3.0.0`](https://github.com/jamestalmage/supports-hyperlinks/releases/tag/v3.0.0).